rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -123,20 +123,33 @@ <h1 class="col-sm-10 col-md-11">{{ record.title }}</h1>
         {{ dl(_('Language'), record.language|language_format(current_i18n.language)) }}
         {% endif %}
 
-
-        {% if record.electronic_location|length > 0 %}
-        {% set uris = [] %}
-        {% for resource in record.electronic_location %}
-        {% do uris.append('<a href="'+ resource.uri +'"><i class="fa fa-lock fa-fw"></i> '+ resource.uri +'</a>') %}
-        {% endfor %}
-        {{ dl(_('Online Access'), uris|join(', ')) }}
+        <!-- Other ACCESS -->
+        {% set other_accesses = record.electronicLocator|get_other_accesses %}
+        {% if other_accesses|length > 0 %}
+        <dt class="col-sm-3 offset-sm-2 offset-md-1">
+          {{ _('Other Access') }}:
```

Comment:
In the mockup, it's written : Title of the fields: "Related resource" for this one

Reply:
ok